onnx/common/path.cc

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/common/path.h"
+#include "onnx/string_utils.h"
+
+namespace ONNX_NAMESPACE {
+
+const std::string path_join(const std::string& origin, const std::string& append) {
```

Comment:
why return const string? this is a utility method to join to strings

Reply:
can this mtd be inlined?

Reply:
Remove const function makes more sense. 
It cannot be inlined.